pkgs/by-name/gl/globalplatform/package.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pcsclite,
+  PCSC,
+  pkg-config,
+  cmake,
+  zlib,
+  pandoc,
+  doxygen,
+  graphviz,
+  openssl,
+  cmocka,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "globalplatform";
+  version = "2.4.0-unreleased";
```

Comment:
From https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning:

> - If a package is a commit from a repository without a version assigned, then the `version` attribute _should_ be the latest upstream version preceding that commit, followed by `-unstable-` and the date of the (fetched) commit. The date _must_ be in `"YYYY-MM-DD"` format.

Reply:
Okay , thank you - I have adjusted the version.